File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/image_generation_request.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ImageGenerationRequest:
    """Image edit request"""
    model: str = None
    prompt: str = None
    n: int = None
    quality: str = None
    size: str = None
    style: str = None
    user: str = None
    image: List[str] = None
    extra: Dict[str, Any] = None
    response_format: str = None

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/ImageGenerationRequest.kt
package com.sdkwork.ai

data class ImageGenerationRequest(
    val model: String? = null,
    val prompt: String? = null,
    val n: Int? = null,
    val quality: String? = null,
    val size: String? = null,
    val style: String? = null,
    val user: String? = null,
    val image: List<String>? = null,
    val extra: Map<String, Any>? = null,
    val responseFormat: String? = null
)
